commons-numbers-fraction/src/test/java/org/apache/commons/numbers/fraction/CommonsLangPortedFractionTest.java

```diff
@@ -0,0 +1,1110 @@
+//
+//        assertEquals(1, Fraction.ONE_THIRD.getNumerator());
+//        assertEquals(3, Fraction.ONE_THIRD.getDenominator());
+//
+//        assertEquals(2, Fraction.TWO_THIRDS.getNumerator());
+//        assertEquals(3, Fraction.TWO_THIRDS.getDenominator());
+//
+//        assertEquals(1, Fraction.ONE_QUARTER.getNumerator());
+//        assertEquals(4, Fraction.ONE_QUARTER.getDenominator());
+//
+//        assertEquals(1, Fraction.TWO_QUARTERS.getNumerator());
+//        assertEquals(2, Fraction.TWO_QUARTERS.getDenominator());
+//
+//        assertEquals(3, Fraction.THREE_QUARTERS.getNumerator());
+//        assertEquals(4, Fraction.THREE_QUARTERS.getDenominator());
+//
+//        assertEquals(1, Fraction.ONE_FIFTH.getNumerator());
+//        assertEquals(5, Fraction.ONE_FIFTH.getDenominator());
+//
+//        assertEquals(2, Fraction.TWO_FIFTHS.getNumerator());
+//        assertEquals(5, Fraction.TWO_FIFTHS.getDenominator());
+//
+//        assertEquals(3, Fraction.THREE_FIFTHS.getNumerator());
+//        assertEquals(5, Fraction.THREE_FIFTHS.getDenominator());
+//
+//        assertEquals(4, Fraction.FOUR_FIFTHS.getNumerator());
+//        assertEquals(5, Fraction.FOUR_FIFTHS.getDenominator());
+    }
+
+    @Test
+    public void testFactory_int_int() {
+        Fraction f = null;
+
+        // zero
+        f = Fraction.of(0, 1);
+        assertEquals(0, f.getNumerator());
+        assertEquals(1, f.getDenominator());
+
+        f = Fraction.of(0, 2);
+        assertEquals(0, f.getNumerator());
+        assertEquals(1, f.getDenominator());
+
+        // normal
+        f = Fraction.of(1, 1);
+        assertEquals(1, f.getNumerator());
+        assertEquals(1, f.getDenominator());
+
+        f = Fraction.of(2, 1);
+        assertEquals(2, f.getNumerator());
+        assertEquals(1, f.getDenominator());
+
+        f = Fraction.of(23, 345);
+        assertEquals(1, f.getNumerator());
+        assertEquals(15, f.getDenominator());
+
+        // improper
+        f = Fraction.of(22, 7);
+        assertEquals(22, f.getNumerator());
+        assertEquals(7, f.getDenominator());
+
+        // negatives
+        f = Fraction.of(-6, 10);
+        assertEquals(-3, f.getNumerator());
+        assertEquals(5, f.getDenominator());
+
+        f = Fraction.of(6, -10);
+        assertEquals(3, f.getNumerator());
+        assertEquals(-5, f.getDenominator());
+
+        f = Fraction.of(-6, -10);
+        assertEquals(-3, f.getNumerator());
+        assertEquals(-5, f.getDenominator());
+
+        // zero denominator
+        assertThrows(ArithmeticException.class, () -> Fraction.of(1, 0));
+        assertThrows(ArithmeticException.class, () -> Fraction.of(2, 0));
+        assertThrows(ArithmeticException.class, () -> Fraction.of(-3, 0));
+
+        // very large: can't represent as unsimplified fraction, although
+        Fraction.of(4, Integer.MIN_VALUE);
+        Fraction.of(1, Integer.MIN_VALUE);
+    }
+
+//    @Test
```

Comment:
Remove if these are not to be supported.

Reply:
@aherbert done.